README.md

```diff
@@ -13,7 +13,10 @@ Azure Cosmos DB is a globally distributed multi-model database. One of the suppo
 	* [Node.js](https://nodejs.org/en/) version v0.10.29 or higher.
 	* [Git](http://git-scm.com/).
 
-2. Clone this repository using `git clone git@github.com:arramac/azure-cosmosdb-documentdb-nodejs-getting-started.git`
+
+1. Clone this repository using `git clone git@github.com:Azure-Samples/azure-cosmos-db-documentdb-nodejs-getting-started.git`

```

Comment:
suggest ...

1. Clone this repository using `git clone git@github.com:Azure-Samples/azure-cosmos-db-documentdb-nodejs-getting-started.git cosmosdb`